Index: gst-plugins-ugly-0.10.18/autogen.sh
===================================================================
--- gst-plugins-ugly-0.10.18.orig/autogen.sh	2011-09-15 16:48:52.668952470 +0900
+++ gst-plugins-ugly-0.10.18/autogen.sh	2011-09-15 16:49:00.112951581 +0900
@@ -30,7 +30,7 @@
 fi
 
 
-CONFIGURE_DEF_OPT='--enable-maintainer-mode --enable-gtk-doc'
+CONFIGURE_DEF_OPT='--enable-maintainer-mode'
 
 autogen_options $@
 
